.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - tracing-[0-9]+.*
+      - tracing-[a-z]+-[0-9]+.*
+
+jobs:
+  create-release:
+    name: Create GitHub release
+    # only publish from the origin repository
+    if: github.repository_owner == 'tokio-rs'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: taiki-e/create-gh-release-action@v1.3.0
+        with:
+          prefix: tracing[\-a-z]*
```

Comment:
bash's `=~` doesn't seem to handle escaped `-` well, so you need to write the following: `tracing[a-z-]*`

Reply:
Well, the handling of hyphens within a prefix is complicated (https://github.com/taiki-e/create-gh-release-action/pull/10), so perhaps the following is more robust: `tracing(-[a-z]+)?`